Added caping of garage spaces

hellomynameislinda opened the pull request:
Added a cap to garage spaces, a function to check availability and implemented this on Index, Park and _Layout views.

Commit:
Added function to check availability in garage.
Added checks on park and index views.

Commit:
Added garage availablility check  to _Layout to disable link to park view if garage is full.

jarethholt commented:
There's another requirement: "Besökare skall informeras om hur många lediga platser det finns i garaget direkt på landningssidan när de besöker applikationen". I think it should be fixable if SpaceAvailable returns an `int` instead and the `if`-statements in the Views check for `SpaceAvailable > 0`.

Commit:
Created a service for garage space availability, moved function from controller. Added necessery code to for injection.

Also made some minor changes to Reciept:
Removed some swedish text.
Removed the _PrintLayout-file and added some css  in the regular _Layout file instead, to hide nav menu and footer on print.

hellomynameislinda commented:
UPDATE:
Created a service for garage space availability to fix a previous bug, moved function from controller. Added necessery code to for injection. Changed function to return int and display the number of available spaces.

Also made some minor changes to Reciept:
Removed some swedish text.
Removed the _PrintLayout-file and added some css  in the regular _Layout file instead, to hide nav menu and footer on print.

Commit:
Changed cap value to 10.

jarethholt reviewed (approved):
Thorough addition to the original PR!

jarethholt committed:
Merge branch 'main' into CapVehiclePositions
